docs/config/QUnit.extend.md

```diff
@@ -34,9 +34,9 @@ QUnit.test( "QUnit.extend", function( assert ) {
     z: undefined
   } );
 
-  assert.equal( base.a, 1, "Unspecified values are not modified" );
-  assert.equal( base.b, 2.5, "Existing values are updated" );
-  assert.equal( base.c, 3, "New values are defined" );
-  assert.ok( !( "z" in base ), "Values specified as `undefined` are removed" );
+  assert.strictEqual( base.a, 1, "Unspecified values are not modified" );
+  assert.strictEqual( base.b, 2.5, "Existing values are updated" );
+  assert.strictEqual( base.c, 3, "New values are defined" );
+  assert.strictEqual( base.z, undefined, "Values specified as `undefined` are removed" );
```

Comment:
This change is in my opinion not correct. 

The first assertions demonstrate that values can be updated or replaced.

This last assertion demonstrates that a property can be deleted. An existing property with the value `undefined` and a property not existing, are two different things, which is why this assertion was here.

```js
var x = {
  foo: undefined
};

'foo' in x; // true, yes - it does exist as local or inherited property.
Object.hasOwnProperty(x, 'foo'); // does true, yes, it exists as local property.
x.foo === undefined; // true, yes it explicitly yields the undefined value.

'foo' in x; // false, property does not exist at all.
Object.hasOwnProperty(x, 'foo'); // false, property does not exist locally.
x.bar === undefined; // true, it implicitly yields the undefined value.
```

I suggest using `assert.false( "z" in base, … )`.